### PR TITLE
Feature a

### PR DESCRIPTION
**Pull request by hejcz**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/hejcz/TravisTest/pull/1" target="_blank">ReviewNinja

File: build.gradle [hunk 1]
```diff
@@ -1,4 +1,8 @@
 apply plugin: 'java'
+apply plugin: 'checkstyle'
+apply plugin: 'pmd'
```
Comment (hejcz) [inline build.gradle]: what does it do?

Comment (hejcz) [inline build.gradle]: !fix this shit

File: build.gradle [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+apply plugin: 'findbugs'
```
Comment (hejcz) [inline build.gradle]: !fixed that stuff

File: build.gradle (hunk 1 above)
Comment (hejcz) [inline build.gradle]: !fixed as well

**Pull request merged**